packages/validation/src/pluginLoaders/PluginLoader.ts

```diff
@@ -60,6 +61,10 @@ export class DefaultPluginLoader implements PluginLoader {
       return new OpenPolicyAgentValidator(parser, wasmLoader);
     });
 
+    this.register('admission-policy', ({parser}) => {
+      return new AdmissionPolicyValidator(parser);
```

Comment:
I wonder how big this module is given that it contains Pako. On [NPM](https://www.npmjs.com/package/pako) it mentions its 1.64MB. Maybe we can dynamically import this module here so that it gets codesplit.